spec/features/search_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe "Search Form" do
+    it "when I fill in the search form with 'sweet potatoes'
+    and I click 'search'
+    then I should be on the page '/foods'
+    then I should see a total of the number of items returned by the search
+    then I should see a list of TEN foods that contain that ingredient
+    and for each of the foods I should see:
+    - the food's GTIN/UPC code
+    - the food's description
+    - the food's Brand Owner
+    - the food's ingredients" do 
+        visit root_path
+        fill_in 'q', with: 'sweet potatoes'
+        click_button('Search')
+        expect(current_path).to eq(foods_path)
+        
+        expect(page).to have_content("Total Number of Items Returned by Search:")
+        expect(page).to have_content("GTIN UPC:")
```

Comment:
It might be a good idea to put these expect for attributes inside of a `within block`. It could look like:
```
within(first('.foods')) do
      expect(page).to have_css(".gtinUpc")
      expect(page).to have_css(".description")
      expect(page).to have_css(".brandOwner")
      expect(page).to have_css(".ingredients")
end
```
We would also want to test that we should be getting 10 results back.